zone/bot.cpp

```diff
@@ -3108,6 +3545,22 @@ bool Bot::Spawn(Client* botCharacterOwner) {
 		SentPositionPacket(0.0f, 0.0f, 0.0f, 0.0f, 0);
 		ping_timer.Start(8000);
 		auto_save_timer.Start(RuleI(Bots, AutosaveIntervalSeconds) * 1000);
+
+		pDontHealMeBefore = 0;
+		pDontGroupHealMeBefore = 0;
+		pDontGroupHoTHealMeBefore = 0;
+		pDontBuffMeBefore = Timer::GetCurrentTime() + 400;
+		pDontDotMeBefore = 0;
+		pDontRootMeBefore = 0;
+		pDontSnareMeBefore = 0;
+		pDontCureMeBefore = 0;
+		pDontRegularHealMeBefore = 0;
+		pDontVeryFastHealMeBefore = 0;
+		pDontFastHealMeBefore = 0;
+		pDontCompleteHealMeBefore = 0;
+		pDontGroupCompleteHealMeBefore = 0;
+		pDontHotHealMeBefore = 0;
```

Comment:
I know this is copying previous patterns but modern practice for our member variables is `snake_case` and `m_variable_name` protected or private